### DISCUSSION TITLE
Guide for creating plugins

### DISCUSSION THREAD

**Discussion by MatthewReed303** [frangoteam/FUXA#511] [General]
Hi @unocelli just wondering when you have some time available if you could do a quick guide on how we can create plugins or integrating node packages. Just don't know where to start to help contribute to the development of the project. Maybe add a developers guide in the docs/wiki and how everything is structured. Thanks

**Comment by jay-jjwu** [comment]
same question, is there any developer guide that for 3rd party plug-in integration?  @unocelli

**Comment by unocelli** [comment]
Hi, currently the project does not have the structure to be able to integrate plug-ins like node-red.

**Comment by RobertBudgen** [comment]
I also would like a basic Modification Guide, I would like add Graphite /carbon as a DataBase option. Basically cloning the influx1.8 configuration and modifying the post format.  and configuration dialog.

**Reply by unocelli** [reply]
Hi, you are welcome...perfect, don't forget to share it
